### ISSUE TITLE
Licence? Can I use this as a starting point?

### ISSUE THREAD

**Issue by multimulti** [ma11hew28/loversapp.com#1]
Hey! I'm about to start a Sinatra facebook app myself and just found your code here.

My app will be quite different (different idea, different design), but there are of course parts we have in common and so I ask for your permission to clone your git and use it as a starting point for my own app.

Of course I would push you some bugfixes I may find in your project, but I'm planning to develop my game closed-source at least until it's well established (or completely failed ;-) ) as I'm hoping to cash out some credits for myself.

Anyway thanks for sharing!

**Comment by ma11hew28** [comment]
Of course! Feel free to fork it and add the MIT license to the root of the project in a file called MIT_LICENSE.md, copyright Acani. Then, send me a pull request.

Thanks!

Matt

**Comment by multimulti** [comment]
Thanks and done :)
